src/arviz_plots/plots/parallel_plot.py

```diff
@@ -0,0 +1,229 @@
+        >>> from arviz_base import load_arviz_data
+        >>> dt = load_arviz_data('centered_eight')
+        >>> plot_parallel(
+        >>>     dt,
+        >>>     var_names=["mu", "tau"],
+        >>> )
+
+    .. minigallery:: plot_parallel
+
+    """
+    if sample_dims is None:
+        sample_dims = rcParams["data.sample_dims"]
+    if isinstance(sample_dims, str):
+        sample_dims = [sample_dims]
+    if visuals is None:
+        visuals = {}
+    if pc_kwargs is None:
+        pc_kwargs = {}
+    else:
+        pc_kwargs = pc_kwargs.copy()
+
+    if backend is None:
+        if plot_collection is None:
+            backend = rcParams["plot.backend"]
+        else:
+            backend = plot_collection.backend
+
+    if labeller is None:
+        labeller = BaseLabeller()
+
+    data = process_group_variables_coords(
+        dt,
+        group=group,
+        var_names=var_names,
+        filter_vars=filter_vars,
+        coords=coords,
+        allow_dict=False,
+    )
+    if len(sample_dims) > 1:
+        data = data.stack(sample=sample_dims)
+        combined_dim = "sample"
+    else:
+        combined_dim = sample_dims[0]
+
+    # create divergence mask
+    sample_stats = get_group(dt, "sample_stats", allow_missing=True)
+    if (
+        sample_stats is not None
+        and "diverging" in sample_stats.data_vars
+        and np.any(sample_stats.diverging)
+    ):
+        div_mask = sample_stats.diverging
+        if coords is not None:
+            div_mask = div_mask.sel(
+                {key: value for key, value in coords.items() if key in div_mask.dims}
+            )
+        if len(sample_dims) > 1:
+            div_mask = div_mask.stack(sample=sample_dims)
+    else:
+        div_mask = xr.zeros_like(data.coords[combined_dim], dtype=bool)
+    data = data.assign_coords(diverging=div_mask)
+    data = dataset_to_dataarray(
+        data, labeller=labeller, sample_dims=[combined_dim], label_type="vert"
+    )
+    if len(data.coords["label"].values) <= 1 or data.ndim != 2:
+        raise ValueError(
+            "Parallel plot requires at least two variables to plot. "
+            "Please provide more than one variable."
+        )
+
+    # get labels and x-values
+    x_labels = data.coords["label"].values
+    x_values = np.arange(len(x_labels))
+
+    new_sample_dims = [combined_dim, "label"]
+    plot_bknd = import_module(f".backend.{backend}", package="arviz_plots")
+    if plot_collection is None:
+        pc_kwargs.setdefault("cols", None)
+        pc_kwargs["figure_kwargs"] = pc_kwargs.get("figure_kwargs", {}).copy()
+        pc_kwargs = set_wrap_layout(pc_kwargs, plot_bknd, data)
+        pc_kwargs["aes"] = pc_kwargs.get("aes", {}).copy()
+        colors = plot_bknd.get_default_aes("color", 1, {})
+        pc_kwargs["aes"].setdefault("color", ["diverging"])
+        pc_kwargs["aes"].setdefault("alpha", ["diverging"])
+        pc_kwargs["color"] = pc_kwargs.get("color", [colors[0], "black"])
```

Comment:
I am unsure about which/how to choose colors. The version in current arviz uses the opposed idea: non diverging samples are black, diverging are "C0". I think I prefer adding the extra emphasis to diverging samples.. But I think C0->non diverging and C1->diverging could also look good. I will try to play around next week and see if there is an alternative I like more, but if someone else has opinions on the matter please comment too.